tlf.1.in

```diff
@@ -1656,6 +1656,11 @@ The station callsign used in messages; also used to determine the station's
 country, zone and continent.
 .
 .TP
+\fBOPERATING_MODE\fR=\fICQ\fR
```

Comment:
add S&P as possible value, see e.g. the description of UNIQUE_CALL_MULTI

Reply:
ok, will do that. Sorry it's first time I'm writing man pages. Syntax is still little bit confusing for me

Reply:
no problem, thanks.